Python/Weather-teller/README.md

```diff
@@ -0,0 +1,41 @@
+# Weather-teller
+Introduction  
+This project , Asks the user the name of a city as input and provides the weather and temperature(in celsius) as output.
+
+Tech stack 
+
+The project is made using python and weather API.
+The weather API , I hv used in this project is from - https://openweathermap.org/api
+
+Demonstration video - here is a view of the project :
+
+https://user-images.githubusercontent.com/91176055/159108679-8124cd46-7467-465a-9674-c32e42361ed7.mp4
+
+
+How to Run locally
+
+Latest version of python must be installed on your local machine.
+
+'requests' package should also be installed.
+
+Pycharm IDE is required.
+
+Clone this repository:
+
+git clone https://github.com/abhijeet007rocks8/Dev-Scripts.git
+
+Change directory:
+
+cd Python
+
+cd Weather-teller
+
+Find main.py and run it.
+
+
+
+If any further information contributors want to include, you are most welcome do.
```

Comment:
Remove this line, this line doesn't required to be included.

if possible then add this section in these manner it looks in Presentable manner this also enhance your PR Skills . 

Add this two sections (install dependencies and Quick Start) in [Readme.md](http://readme.md/) file in these particlular manner : 
<img width="1100" alt="Screenshot 2022-03-03 at 1 02 07 PM" src="https://user-images.githubusercontent.com/78725162/158399068-9c84a79b-17d7-47cc-b1ec-02ee536cfcd2.png">
<br />
**Quick Start** (how to run in local)
<hr />
<img width="1080" alt="Screenshot 2022-03-03 at 1 01 42 PM" src="https://user-images.githubusercontent.com/78725162/158399122-5fcc235c-b2af-4607-ac48-5e97b9481f4b.png">